app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Details Page for a Specific User</h1>
+<section>
+  <%= render "components/user-card", user: @user  %>
+  <div class="bio-wrapper">
+    <p><%= @user.bio %></p>
+  </div>
+  <%if @user.posts.any? %>
```

Comment:
- [ ] As mentioned in the project requirements, please use the last_three_posts` method you created in your model to render the last 3 posts.